include/macros.h

```diff
@@ -96,6 +96,7 @@ extern GraphicsContext* __gfxCtx;
         s32 __dispPad
 
 #define CLOSE_DISPS(gfxCtx) \
+    (void)0;                \
```

Comment:
Would it be better to to this in the one place that it's needed than change the macro?

Reply:
The idea is to make it so you can use the macro without context, and to have the same behavior as the debug version, since the debug version has a function call which acts as the valid statement before the bracket.
Also it's probably going to be needed in more places because using a goto to CLOSE_DISPS is the "right" way to do an early return in a draw function. So I think we might as well add it to the macro now and avoid more headache in the future.

Reply:
I see, makes sense.